src/sentry/api/endpoints/project_details.py

```diff
@@ -663,6 +674,25 @@ def put(self, request: Request, project) -> Response:
                 changed_proj_settings["sentry:relay_pii_config"] = (
                     result["relayPiiConfig"].strip() or None
                 )
+        if "relayCustomMetricCardinalityLimit" in result:
+            limit = result.get("relayCustomMetricCardinalityLimit")
+            cardinality_limits = []
+            if limit is not None:
+                # For now we only allow setting a single limit
+                # TODO: validate this with rust validator
```

Comment:
lets create a task somewhere for this and follow up

Reply:
https://github.com/getsentry/sentry/issues/69750